test/Microsoft.AspNet.Mvc.ViewFeatures.Test/ViewFeatures/SessionStateTempDataProviderTest.cs

```diff
@@ -16,52 +16,45 @@ namespace Microsoft.AspNet.Mvc.ViewFeatures
     public class SessionStateTempDataProviderTest
     {
         [Fact]
-        public void Load_NullSession_ReturnsEmptyDictionary()
+        public void Load_ThrowsException_WhenSessionIsNotEnabled()
         {
             // Arrange
             var testProvider = new SessionStateTempDataProvider();
 
-            // Act
-            var tempDataDictionary = testProvider.LoadTempData(
-                GetHttpContext(session: null, sessionEnabled: true));
-
-            // Assert
-            Assert.Null(tempDataDictionary);
+            // Act & Assert
+            var exception = Assert.Throws<InvalidOperationException>(() =>
```

Comment:
If not, remove the unused `exception` variable. Similar changes for other tests.